assets/scripts/esDevelopersItaliaQuery.js

```diff
@@ -64,10 +64,12 @@ function esDevelopersItaliaManager(queryType, config, objectConfig, params) {
 
     // IDs of DOM elements containing input checkboxes to use
     'filterKeys': {
+      'publiccode.intendedAudience.scope': 'list-tags',
       'publiccode.categories': 'list-tags',
       'publiccode.developmentStatus': 'list-status'
     },
     'filterKeysMobileSelectors': {
+      'publiccode.intendedAudience.scope  ': 'pills-arguments',
```

Comment:
```suggestion
      'publiccode.intendedAudience.scope': 'pills-arguments',
```
extra spaces needed?

Reply:
nope